services/algolia-sync/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "@base-cms/algolia-sync",
+  "version": "1.0.0",
+  "main": "src/index.js",
+  "author": "Jeremy Laird <jlaird@endeavorb2b.com>",
+  "repository": "https://github.com/base-cms/base-cms/tree/master/services/algolia-sync",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "dev": "node src/index.js",
+    "lint": "eslint --ext .js --max-warnings 5 ./",
+    "test": "yarn lint"
+  },
+  "dependencies": {
+    "@base-cms/graphql-fragment-types": "^1.9.0",
+    "algoliasearch": "^4.0.0",
+    "apollo-cache-inmemory": "^1.6.5",
+    "apollo-client": "^2.6.8",
+    "apollo-link-http": "^1.5.16",
+    "aws-sdk": "^2.610.0",
+    "cli-progress": "^3.6.0",
+    "dotenv": "^8.2.0",
```

```suggestion
    "@base-cms/env": "^1.0.0",
```
This monorepo uses `envalid` as provided by the `@base-cms/env` sub-package. You'll need to define your required env vars in a new `env.js` file inside your service folder -- check the `graphql-server` service for an example.